core/src/main/scala/org/apache/spark/ui/exec/ExecutorThreadDumpPage.scala

```diff
@@ -128,8 +128,15 @@ private[ui] class ExecutorThreadDumpPage(
 
   // scalastyle:off
   private def drawExecutorFlamegraph(request: HttpServletRequest, thread: Array[ThreadStackTrace]): Seq[Node] = {
+    val js =
+      s"""
+         |import {drawFlamegraph} from "/static/flamegraph.js";
```

Comment:
Hmmm.. shouldn't we fix it as:

```scala
import {drawFlamegraph} from "${UIUtils.prependBaseUri(request, "/static/flamegraph.js")}";
```

? this and all those imports in this PR. Otherwise, I think it's a regression of SPARK-11484 and when a proxy is used.

Reply:
It makes sense to me, let me fix this. Thank you, Hyukjin.